setup.cfg

```diff
@@ -22,6 +22,7 @@ install_requires =
     watchdog
     pyyaml
     intervaltree
+    GitPython
```

Comment:
Not for this PR but we should probably try split into a a few section of requirements

Reply:
removed as the cgi hook is not a python file any more